AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageDetectionViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  AutomaticLanguageDetectionViewController.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AmazonChimeSDK
+import UIKit
+
+class AutomaticLanguageDetectionViewController: UIViewController, UITextFieldDelegate {
+    
+    @IBOutlet var languageDetectionTableView: UITableView!
+    @IBOutlet var preferredLanguagesTextField: UITextField!
+    var preferredLanguagesPickerView = UIPickerView()
+    public var cancellationHandler: ((Int?) -> Void)?
+    public var selectedLanguageOptionsHandler: ((String?) -> Void)?
+    public var preferredLanguageOptionHandler: ((String?) -> Void)?
+    public var languages: [String] = []
+    public var languagesDict: [String: String] = [:]
+    
+    var preferredLanguages = ["-- Optional --"]
+    var selectedLanguages = " "
+    var preferredLanguageSelected = " "
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.preferredLanguagesPickerView.delegate = self
+        self.preferredLanguagesPickerView.dataSource = self
+        self.preferredLanguagesTextField.delegate = self
+        self.preferredLanguagesTextField.inputView = preferredLanguagesPickerView
+        preferredLanguagesTextField.text = preferredLanguages[0]
+    }
+    
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        return false
+    }
+    
+    @IBAction func saveLanguageDetectionOptions(_ sender: Any) {
+        if preferredLanguages.count < 3 {
+            self.view.makeToast("Must select at least two languages!")
+            return
+        } else {
+            
+            let selectedIndexPaths = getSelectedIndexPaths()
+            selectedLanguages = ""
+            for index in selectedIndexPaths.indices.dropLast() {
```

Comment:
sry why we do drop last?

Reply:
when we start the transcription with automatic language detection, it expects a list of languages like 
`fr-FR,en-US,es-US` etc. I don't want to add a comma after the last selected language, so the first n-1 selected languages are `l1, l2, l3,...,` then the last language is added without the comma.